src/events/attributes.rs

```diff
@@ -57,18 +57,11 @@ impl<'a> Attribute<'a> {
     /// # Pre-condition
     ///
     /// The keys and values of `custom_entities`, if any, must be valid UTF-8.
```

Comment:
Do not forget to update documentation on all methods

Reply:
Right, it's just a draft, looking mostly for technical feedback